docs/framework/wcf/how-to-create-a-wcf-client.md

```diff
@@ -47,7 +47,7 @@ The client application uses the generated proxy class to communicate with the se
     ```
 
     ```shell
-    svcutil.exe /language:cs /out:generatedProxy.cs /config:app.config http://localhost:8000/GettingStartedClient/service
+    svcutil.exe /language:cs /out:generatedProxy.cs /config:app.config http://localhost:8000/GettingStarted/CalculatorService
```

Comment:
It would be great if you could fix the intro URL also. It should not use `GettingStartedClient`